test/modules/events.js

```diff
@@ -499,9 +499,9 @@
             assessGestureExpectations({
                 description:           'enter-nonprimarypress-nonprimaryrelease-exit (secondary/right click):  ',
                 enterCount:            1,
-                exitCount:             1,
+                exitCount:             2,
```

Comment:
abortContacts simulates the exit event, so after the non-primary button press with the mouse, it tries to release the mouse button and removes the gesture points for it. Prior to this change, I had it checking for contacts > 0, but abortContacts operates based on whether the pointer has any gesture points stored, which the mouse has from simply hovering. This was the case where I was most concerned about possible side effects, but I didn't see any issues when I tested--the next mousemove event restores the state. But now I'm wondering if I should add the contacts > 0 check to abortContacts for just mouse, or for mouse and pen--leaving out touch since touch can't hover.